### PR TITLE
Fix/workflow and ct gf overwrite fix

### PR DESCRIPTION
**Pull request by nadeem-cs**

### CODE REVIEW

File: lib/stack/contentType/index.js [hunk 1]
```diff
@@ -192,17 +192,18 @@ export function ContentType (http, data = {}) {
...
-    this.import = async function (data) {
+    this.import = async function (data, params = {}) {
       try {
         const response = await upload({
           http: http,
           urlPath: `${this.urlPath}/import`,
           stackHeaders: this.stackHeaders,
-          formData: createFormData(data)
+          formData: createFormData(data),
+          params: params
```
Comment (ishaileshmishra) [inline lib/stack/contentType/index.js:206]: Please add missing tests as well @nadeem-cs

File: lib/stack/globalField/index.js [hunk 2]
```diff
@@ -115,17 +115,18 @@ export function GlobalField (http, data = {}) {
...
-    this.import = async function (data) {
+    this.import = async function (data, params = {}) {
       try {
         const response = await upload({
           http: http,
           urlPath: `${this.urlPath}/import`,
           stackHeaders: this.stackHeaders,
-          formData: createFormData(data)
+          formData: createFormData(data),
+          params: params
```
Comment (ishaileshmishra) [inline lib/stack/globalField/index.js:129]: Please add missing tests as well @nadeem-cs

**Comment by nadeem-cs** [comment]
Resolved the conversations. Pls check and provide approval @ishaileshmishra

**Review by ishaileshmishra — APPROVED** [review]
Approved

**Pull request merged**